app/lib/context-helper.js

```diff
@@ -0,0 +1,79 @@
+const { getAllApplicationsBySbi } = require('../api-requests/application-service-api')
+const { isWithin10Months } = require('./date-utils')
+const session = require('../session')
+const {
+  endemicsClaim: {
+    latestEndemicsApplication: latestEndemicsApplicationKey,
+    latestVetVisitApplication: latestVetVisitApplicationKey,
+    previousClaims: previousClaimsKey
+  }
+} = require('../session/keys')
+const { getClaimsByApplicationReference } = require('../api-requests/claim-service-api')
+const { getEndemicsClaim } = require('../session')
+const { claimType } = require('../constants/claim')
+
+async function refreshApplications (request) {
+  // get ALL the applications
```

Comment:
I really don't think we need comments like this. The method in the line below is `getAllApplications...`. I've said it before but I do think we should try and follow Defra principles of letting the code explain itself and avoiding comments that are just echoing what the code is obviously doing. Personally I find them distracting when trying to read the code itself.

Reply:
I agree we don't need pointless comments, but a lot of the point of this pR was to try clear up what the actual behaviour is in this app because (in general) it really **isn't** clear from the code itself. Given that I added quite a few comments to assist as I pored through, and this is one where actually yep having changed the function name afterwards this comment became redundant, so I can delete it. However I'm also not sure we should be losing any sleep over an extra comment being there either, as to me it just isn;t the end of the world. i will remove this one though

Reply:
Deleted this one